booster_bdd/features/src/stackAnalyses.py

```diff
@@ -0,0 +1,113 @@
+import requests
+import features.src.support.helpers as helpers
+import re
+import os
+import time
+from urllib.parse import urlparse, urljoin
+
+boosterLaunched = False
+
+class StackAnalyses(object):
+
+    # Obtain the key to use to access the stack analyses information 
+    def getReportKey (self, codebaseUrl):
+        print('Getting Stack Analyses report key.....')
+
+        theToken = helpers.get_user_tokens().split(";")[0]
+        authHeader = 'Bearer {}'.format(theToken)
+
+        # TODO - Convert to environment variables
+        _url = 'https://recommender.api.openshift.io'
+        _endpoint = '/api/v1/stack-analyses'
+
+        headers = {'Accept': 'application/json',
+                   'Authorization': authHeader,
+                   'X-App': 'osio',
+                   'X-Git-Provider': 'GitHub',
+                   'Content-Type': 'application/x-www-form-urlencoded'}
+
+        # Remove the ".git" suffix from the URL
+        # print ('the target URL is:' + codebaseUrl)
+        shortenedUrl = codebaseUrl[:-4]
```

Comment:
I'd add something like: `if shortenedUrl.endswith(".git"):`

Reply:
Good idea! I will add that.